packages/api/lib/asset.interface.ts

```diff
@@ -66,3 +66,14 @@ export interface Action {
 }
 
 export type ActionRevision = Action & { originalId: string };
+
+export interface EventCause {
+  cause: string;
+  level: string;
+}
+
+export interface EventLevelOverride {
```

Comment:
gonna change wording in hpc repo too

Reply:
define interfaces here or in event.interface.ts?

Reply:
it is fine here. the structure is asset.service.ts -> asset.interface.ts. as there is no event service file this can stay here.